src/Tools/AnalyzerRunner/FixAllContextHelper.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace AnalyzerRunner
+{
+    internal static class FixAllContextHelper
+    {
+        public static async Task<ImmutableDictionary<Document, ImmutableArray<Diagnostic>>> GetDocumentDiagnosticsToFixAsync(FixAllContext fixAllContext)
```

Comment:
same here.

Reply:
if we want to have it but not currently used, can we put #if false, #endif to let previewers know it is currently not used.